tests/unit/resources/servers/test_id_pools_ipv4_subnets.py

```diff
@@ -31,38 +31,48 @@ def setUp(self):
         self.client = IdPoolsIpv4Subnets(self.connection)
         self.example_uri = "/rest/id-pools/ipv4/subnets/f0a0a113-ec97-41b4-83ce-d7c92b900e7c"
 
-    @mock.patch.object(ResourceClient, 'create')
+    @mock.patch.object(ResourceHelper, 'create')
```

Comment:
since you have inherited create method from Resource class, use the same class here.